rust/Makefile

```diff
@@ -62,6 +63,10 @@ $(objtree)/rust/exports_kernel_generated.h: exports_target_type := _RUST_GPL
 $(objtree)/rust/exports_kernel_generated.h: $(objtree)/rust/kernel.o FORCE
 	$(call if_changed,exports)
 
+$(objtree)/rust/exports_compiler_builtins_generated.h: exports_target_type := _RUST
+$(objtree)/rust/exports_compiler_builtins_generated.h: $(objtree)/rust/compiler_builtins.o FORCE
+	$(call if_changed,exports)
+
```

Comment:
Can you please move these in-between `core` and `alloc`? (I keep them like that due to the dependency chain).